content/operate/oss_and_stack/management/security/acl.md

```diff
@@ -293,16 +283,19 @@ The following is a list of command categories and their meanings:
 * **bitmap** - Data type: bitmaps related.
 * **blocking** - Potentially blocking the connection until released by another
   command.
+* **bloom** - Data type: all Bloom filter related commands.
+* **cms** - Data type: count-min sketch related commands.
 * **connection** - Commands affecting the connection or other connections.
   This includes [`AUTH`](/commands/auth), [`SELECT`](/commands/select), [`COMMAND`](/commands/command), [`CLIENT`](/commands/client), [`ECHO`](/commands/echo), [`PING`](/commands/ping), etc.
+* **cuckoo** - Data type: all Cuckoo filter related commands.
 * **dangerous** - Potentially dangerous commands (each should be considered with care for
   various reasons). This includes [`FLUSHALL`](/commands/flushall), [`MIGRATE`](/commands/migrate), [`RESTORE`](/commands/restore), [`SORT`](/commands/sort), [`KEYS`](/commands/keys),
   [`CLIENT`](/commands/client), [`DEBUG`](/commands/debug), [`INFO`](/commands/info), [`CONFIG`](/commands/config), [`SAVE`](/commands/save), [`REPLICAOF`](/commands/replicaof), etc.
+* **fast** - Fast O(1) commands. May loop on the number of arguments, but not the number of elements in the key.
 * **geo** - Data type: geospatial indexes related.
 * **hash** - Data type: hashes related.
 * **hyperloglog** - Data type: hyperloglog related.
-* **fast** - Fast O(1) commands. May loop on the number of arguments, but not the
-  number of elements in the key.
+* **json** - Data type: all JSON related commands.
```

Comment:
I noticed that the wording is different for different data structures, e.g.,

> **hash** - Data type: hashes related.
> **json** - Data type: all JSON related commands.

I prefer the second because ACL rules are based on command groups, not on the Redis key data type.

Reply:
I've updated the other categories to be consistent. Fixed in next push.